.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: rust
 sudo: false
 rust:
-  - 1.22.1
+  - 1.26.0  # MSRV
   - stable
   - beta
```

Comment:
do we need to test both beta and nightly? anything to decrease CI load is good atm